player/lua/osc.lua

```diff
@@ -2058,7 +2058,8 @@ function process_event(source, what)
 
             if n == 0 then
                 --click on background (does not work)
-            elseif n > 0 and not (elements[n].eventresponder[source .. "_" .. what] == nil) then
+            elseif n > 0 and not (n > #elements) and
+                not (elements[n].eventresponder[source .. "_" .. what] == nil) then
```

Comment:
Feel free to change the whole script, I just did it this way because I was too lazy to remember what lua's I-HAVE-TO-BE-A-FUCKING-SPECIAL-SNOWFLAKE-not-equal operator was.